src/index.cpp

```diff
@@ -746,7 +746,7 @@ namespace diskann {
 #ifdef EXEC_ENV_OLS
     _u32 nodes_read = 0;
     _u64 cc = 0;
-    _u64 graph_offset = file_offset + header_size;
```

Comment:
this `file_offset `was undefined, and causes build error. I guess graph_offset starts right after header so removed it.

Reply:
The file offset parameter only makes sense when we have a single file index. This won't apply here, so the change is good.